pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -650,14 +687,33 @@ func (gb *GraphBuilder) processGraphChanges() bool {
 			for _, dep := range potentiallyInvalidDependents {
 				if len(observedIdentity.Namespace) > 0 && dep.identity.Namespace != observedIdentity.Namespace {
 					// Namespace mismatch, this is definitely wrong
-					klog.V(2).Infof("node %s references an owner %s but does not match namespaces", dep.identity, observedIdentity)
+					logger.V(2).Info("node references an owner but does not match namespaces", "apiVersion", event.gvk.GroupVersion().String(),
```

Comment:
This logs a lot more information than before.

At first glance that seems useful, but what about performance implications?

Reply:
I was mostly copying the data from the LoggerWithValues call from above (as is required in this transitional state). If you'd prefer, I can revert back so each Info statement only logs what was previously being logged?

Reply:
Let's log only what was being logged before.